src/com/epam/Exceptions/models/Mark.java

```diff
@@ -0,0 +1,47 @@
+package com.epam.Exceptions.models;
+
+import com.epam.Exceptions.MarkOutOfRangeException;
+
+public class Mark {
+    private int value;
+    private String subject;
+
+    private Mark(MarkBuilder builder) {
+        this.value = builder.value;
+        this.subject = builder.subject;
+    }
+
+    public static class MarkBuilder {
+        private int minMarkValue=0;
```

Comment:
Please create constants for MIN_MARK_VALUE and MAX_MARK_VALUE as described above for Strings

Reply:
Please format the code